lib/beaker-pe/install/pe_utils.rb

```diff
@@ -683,6 +688,41 @@ def simple_monolithic_install(master, agents, opts={})
           end
         end
 
+        # Configure PE on a monolithic master and some number of frictionless agents if a node is using a PE preloaded image for the master node.
+        # @param [Host] master The node to configure the master on
+        # @param [Array<Host>] agents The nodes to install agents on
+        # @param [Hash{Symbol=>Symbol, String}] opts The options for how to install or upgrade PE
+        #
+        # @example
+        #   simple_monolithic_install_with_preload(master, agents, {:type => :install, :pe_ver => '2017.2.0'})
+        #
+        # @return nil
+        #
+        # @api private
+        def simple_monolithic_install_with_preload(master, agents, opts={})
+          step "Performing a standard monolithic install with frictionless agents on a preloaded image"
+          all_hosts = [master, *agents]
+          configure_type_defaults_on([master])
+
+          # Set PE distribution on the agents, creates working directories
+          prepare_hosts(all_hosts, opts)
+          register_feature_flags!(opts)
+          generate_installer_conf_file_for(master, all_hosts, opts)
+
+          step "Stop agent on master" do
+            stop_agent_on(master)
+          end
+
+          if manage_puppet_service?(master[:pe_ver], options)
+            configure_puppet_agent_service(:ensure => 'stopped', :enabled => false)
+          end
+
+          install_agents_only_on(agents, opts)
+
+          step "Run puppet" do
+            on(master, puppet_agent('-t'), :acceptable_exit_codes => [0,2])
```

Comment:
What does this puppet run accomplish?

Reply:
looks like nothing at all, thanks for pointing it out. removing now